models/biocro/R/run.biocro.R

```diff
@@ -138,6 +139,8 @@ run.biocro <- function(lat, lon, metpath, soil.nc = NULL, config = config, coppi
   daily.results <- hourly.results[, list(Stem = max(Stem), 
                                          Leaf = max(Leaf),
                                          Root = max(Root), 
+                                         AboveLitter = max(AboveLitter),
```

Comment:
Canopy transpiration has time in the denominator, precip doesn't, and evaporation it depends which BioCro function's docs I look at :p 

I''ve opened a separate ticket (#1338) to make run.biocro check for hourly input.